test/hotspot/jtreg/runtime/valhalla/inlinetypes/JumboInline.java

```diff
@@ -110,10 +81,9 @@ public boolean equals(Object o) {
             JumboInline j = (JumboInline)o;
             return (l0 == j.l0 && l1 == j.l1 && l2 == j.l2 && l3 == j.l3
                     && l4 == j.l4 && l5 == j.l5 && l6 == j.l6 && l7 == j.l7
-                    && l8 == j.l8 && l9 == j.l9 && l10 == j.l10 && l7 == j.l10
```

Comment:
`l7 == j.l10`? Happens to work, because these are always 0, per the above bug.

Reply:
The newer code here is cleaner, and more importantly correct. Again, agree, it only worked since "0" default